modisco

```diff
@@ -78,6 +81,61 @@ The options are as follows:
 )
 meme_parser.add_argument("-o", "--output", type=str, help="The path to the output file.")
 
+seqlet_bed_parser = subparsers.add_parser("seqlet-bed", help="Create a BED file of seqlets from a modisco results file.")
+seqlet_bed_parser.add_argument("-i", "--h5py", type=str, required=True,
+	help="An HDF5 file containing the output from modiscolite.")
+seqlet_bed_parser.add_argument("-o", "--output", type=str, required=True,
+	help="The path to the output file.")
+seqlet_bed_parser.add_argument("-p", "--peaksfile", type=str, required=True,
+	help="The path to the peaks file. This is to compute the absolute start and\
+end positions of the seqlets within a reference genome, as well as the chroms.")
+seqlet_bed_parser.add_argument("-c", "--chroms", type=str, required=True,
+	help="""A comma-delimited list of chromosomes, or '*', denoting which
+chromosomes to process. Should be the same set of chromosomes used during
+interpretation. Examples: 'chr1,chr2,chrX' || '*' || '1,2,X'.""")
+seqlet_bed_parser.add_argument("-w", "--windowsize", type=int,
+	help="""Optional. This is for backwards compatibility for older modisco h5
+files that don't contain the window size as an attribute. This should be set
+the to size of the window around the peak center that was used for.""")
+
+seqlet_fasta_parser = subparsers.add_parser("seqlet-fasta", help="Create a FASTA file of seqlets from a modisco results file.")
+seqlet_fasta_parser.add_argument("-i", "--h5py", type=str, required=True,
+	help="An HDF5 file containing the output from modiscolite.")
+seqlet_fasta_parser.add_argument("-o", "--output", type=str, required=True,
+	help="The path to the output file.")
+seqlet_fasta_parser.add_argument("-p", "--peaksfile", type=str, required=True,
+	help="The path to the peaks file. This is to compute the absolute start and\
+end positions of the seqlets within a reference genome, as well as the chroms.")
+seqlet_fasta_parser.add_argument("-s", "--sequences", type=str, required=True,
+	help="A .npy or .npz file containing the one-hot encoded sequences.")
+seqlet_fasta_parser.add_argument("-c", "--chroms", type=str, required=True,
+	help="""A comma-delimited list of chromosomes, or '*', denoting which
+chromosomes to process. Should be the same set of chromosomes used during
+interpretation. Examples: 'chr1,chr2,chrX' || '*' || '1,2,X'.""")
+seqlet_fasta_parser.add_argument("-w", "--windowsize", type=int,
+	help="""Optional. This is for backwards compatibility for older modisco h5
+files that don't contain the window size as an attribute. This should be set
+the to size of the window around the peak center that was used for.""")
+
+def convert_arg_chroms_to_list(chroms: str) -> List[str]:
+	"""Converts the chroms argument to a list of chromosomes."""
+	if chroms == "*":
+		# Return all chromosome numbers
+		return [str(i) for i in range(1, 23)] + ["X", "Y"]
```

Comment:
chrY is not always used by default because not all samples will have it. Most are autosomes + X.

Reply:
Done- left out `Y `by default